tests/ignite/metrics/test_psnr.py

```diff
@@ -123,100 +125,160 @@ def update(engine, i):
     assert np.allclose(result, np_psnr / np_y.shape[0], atol=atol)
 
 
-def _test_distrib_integration(device, atol=1e-8):
+def _test_distrib_input_float(device, atol=1e-8):
+    def get_test_cases():
+
+        y_pred = torch.rand(n_iters * batch_size, 2, 2, device=device)
+        y = y_pred * 0.65
+
+        return y_pred, y
 
-    rank = idist.get_rank()
     n_iters = 100
-    s = 10
-    offset = n_iters * s
+    batch_size = 10
 
-    # test for float
-    manual_seed(42)
-    y_pred = torch.rand(offset * idist.get_world_size(), 3, 28, 28, device=device)
-    y = y_pred * 0.65
-    data_range = (y.max() - y.min()).cpu().item()
-    _test(y_pred, y, data_range, "cpu", n_iters, s, offset, rank, atol=atol)
+    rank = idist.get_rank()
+    for i in range(3):
+        # check multiple random inputs as random exact occurencies are rare
+        torch.manual_seed(42 + rank + i)
+        y_pred, y = get_test_cases()
+        _test(
+            y_pred,
+            y,
+            1,
+            "cpu",
+            n_iters,
+            batch_size,
+            atol=atol,
```

Comment:
```suggestion
            atol=atol
```

Reply:
When I eliminate the last comma, but when I run `run_code_style.sh` it reverts.
Let me manually change the line and send again.

Reply:
No, it works. For complicated case with lambda function you can do something like
```python
    def out_fn(x):
        return x[0][:, 0, ...], x[1][:, 0, ...]

    rank = idist.get_rank()
    for i in range(3):
        # check multiple random inputs as random exact occurencies are rare
        torch.manual_seed(42 + rank + i)
        y_pred, y = get_test_cases()
        _test(y_pred, y, 220, "cpu", n_iters, batch_size, atol, output_transform=out_fn, compute_y_channel=True)
        if device.type != "xla":
            dev = idist.device()
            _test(y_pred, y, 220, dev, n_iters, batch_size, atol, output_transform=out_fn, compute_y_channel=True)

```